### ISSUE TITLE
getInitialProps gets called multiple times for AMP (reopen)

### ISSUE THREAD

**Issue by getanwar** [zeit/next.js#9212]
# Bug report
I commented on my previously closed issue (#9111) to reopen that, but it was neither reopened nor responded by anyone. So I had to create a new issue for the same problem I am facing. 😢 

## Describe the bug

Codes in `getInitialProps` get called multiple times if the component is AMP. I tried installing a fresh `nextjs` project and faced the same behavior.

## To Reproduce

```js
export const config = { amp: true };

const Home = () => (
   // ...
);

Home.getInitialProps = () => {
    console.log("This is getting called so many times");
    return {
        a: 1 // return a valid object
    };
};

export default Home;

```

Now open `home` in a browser and look at the terminal. I cleared the terminal and reloaded the page again but the result is same. There are multiple console logs (at least 6 for me!).

## Expected behavior

Should get called only once.

## Screenshots

<img width="402" alt="Screen Shot 2019-10-17 at 5 04 36 PM" src="https://user-images.githubusercontent.com/6163268/67016096-4d425e80-f100-11e9-86ab-e97d4142ada0.png">

## System information

- OS: macOS (Mojave)
- Browser: Chrome (latest)
- Version of Next.js: 9.1.1

## Additional context

There is a warning in the terminal but no message is shown!

### Additional information

If it was only occurred in `dev` then I would have guessed HRM is causing it. But it's happening in `build` also. Here is a screenshot:

<img width="315" alt="Screen Shot 2019-10-17 at 5 10 15 PM" src="https://user-images.githubusercontent.com/6163268/67016623-26385c80-f101-11e9-9ca1-4a2dcf1eb101.png">

### Response taken from my previous issue
<img width="762" alt="Screen Shot 2019-10-27 at 3 48 14 PM" src="https://user-images.githubusercontent.com/6163268/67634835-b6309000-f8d1-11e9-88a5-e5d95d646c8b.png">

**Comment by ijjk** [comment]
The above mentioned reply shows that it was opened in a browser in the attached gif 😕. Your screenshot is running in dev mode and you say above:
> If it was only occurred in dev then I would have guessed HRM is causing it. But it's happening in build also. Here is a screenshot:

it is **not** occurring in production mode which that screenshot shows. I'm going to re-close this since it seems no new details have been provided and the details in the gif might have been missed

**Comment by luizdotpro** [comment]
Exactly the same problem here, but with four calls, not six.

**Comment by getanwar** [comment]
@ijjk It was occurring only in Chrome (Normal Window), I tried in several other browser after your second reply, It was working as expected! I didn't have any clue what might have causing that, cleared all the storages, cookies for `localhost` with no luck. Lastly I reset chrome settings to default. Now it is fine. I have no idea which setting was causing that. Anyone reading this, post if you have an answer. 

Thanks for you time.

**Comment by getanwar** [comment]
FOUND IT! I have Amp Validator Extension installed in Chrome, it was causing the issue. I should have guessed it. @luizdotpro Check if are in the same scenario.

**Comment by luizdotpro** [comment]
Uow! yeah thats it maybe, I have an extension also! 😮  I will test as soon as possible and post here if it solved the rerendering issue.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
